pkg/controller/kieapp/constants/constants.go

```diff
@@ -260,6 +260,23 @@ const (
 	//Envs
 	KubeNS     = "KUBERNETES_NAMESPACE"
 	KubeLabels = "KUBERNETES_LABELS"
+
+	//Kafka extension
+	KIE_SERVER_KAFKA_EXT_DISABLED          = "KIE_SERVER_KAFKA_EXT_DISABLED"
+	KIE_SERVER_KAFKA_EXT_TOPICS            = "KIE_SERVER_KAFKA_EXT_TOPICS"
+	KIE_SERVER_KAFKA_EXT_SERVERS           = "KIE_SERVER_KAFKA_EXT_SERVERS"
+	KIE_SERVER_KAFKA_EXT_CLIENT_ID         = "KIE_SERVER_KAFKA_EXT_CLIENT_ID"
+	KIE_SERVER_KAFKA_EXT_AUTOCREATE_TOPICS = "KIE_SERVER_KAFKA_EXT_AUTOCREATE_TOPICS"
+	KIE_SERVER_KAFKA_EXT_GROUP_ID          = "KIE_SERVER_KAFKA_EXT_GROUP_ID"
+	KIE_SERVER_KAFKA_EXT_ACKS              = "KIE_SERVER_KAFKA_EXT_ACKS"
+	KIE_SERVER_KAFKA_EXT_MAX_BLOCK_MS      = "KIE_SERVER_KAFKA_EXT_MAX_BLOCK_MS"
+
+	//default values
+	KIE_KAFKA_EXT_MAX_BLOCK_MS_DEFAULT      = 2000
```

Comment:
I think the pattern for consts in Golang is camelcase.

Reply:
Ok